docs/source/getting-started/setting-up-driver-plugin.adoc

```diff
@@ -116,11 +116,11 @@ You can verify the existing version of the xhyve driver on your system using:
 
 ----
 $ brew info --installed docker-machine-driver-xhyve
-docker-machine-driver-xhyve: stable 0.3.1 (bottled), HEAD
+docker-machine-driver-xhyve: stable 0.3.3 (bottled), HEAD
 Docker Machine driver for xhyve
 https://github.com/zchee/docker-machine-driver-xhyve
-/usr/local/Cellar/docker-machine-driver-xhyve/0.3.1 (3 files, 13.2M) *
-  Poured from bottle on 2016-12-20 at 17:44:35
+/usr/local/Cellar/docker-machine-driver-xhyve/0.3.1 (3 files, 10.3MB) *
```

Comment:
@anjannath how come it still have `0.3.1`?

Reply:
Yikes, fixing it now, i copy pasted the output of `brew` but there was no new lines, then manually edited it, that got left out.